app/graphql/types/query_type.rb

```diff
@@ -14,6 +14,7 @@ def user(id:)
     end
 
     # Find a list of characters for a single user
+    # Do we need both of these `:user` field declerations or can we consolidate them to one and just use def on both of them?
     field :user, Types::UserType, null: false do
       argument :id, ID, required: true
     end
```

Comment:
Suzie, your expertise ples!

Reply:
oh yeah I think this is just duplicate code. Great catch!